QATCH/ui/mainWindow_ui.py

```
@@ -698,6 +760,166 @@ def setLearnMode(self, obj=None, tab_index=0):
         if obj == None:
             return False
 
+    def setDonnanMode(self, obj=None):
+        if self.splitter.widget(0) == self.donnan_ui and not self._force_splitter_mode_set:
+            Log.d("Donnan-Gibbs Calculator already active. Skipping mode change request.")
+            if obj == None:
```

Ruff (0.15.15)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)